react/components/RenderPage.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react'
+import ExtensionPoint from './ExtensionPoint'
+import MaybeAuth from './MaybeAuth'
+import MaybeContext from './MaybeContext'
+import { RenderContext } from './RenderContext'
+
+interface Props {
+  page: string,
+  query?: Record<string, string>
+}
+
+export default class RenderPage extends PureComponent<Props> {
```

Comment:
I didn't quite get how this will render nested extension points for e.g. admin but _TYP_

Reply:
Ah, got it.